### PR TITLE
Add build and release steps for macOS and linux

### LINKED ISSUE #28: Needs unsupported platform builds/releases
Should be straightforward to build these just like dcam or egrabber.

### PR DESCRIPTION
**Pull request by andy-sweet**
Closes #28

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -43,13 +43,77 @@ jobs:
...
 
-      - name: Package
+      - name: CMake
+        run: cmake -B ${{github.workspace}}/build -DCMAKE_BUILD_TYPE=Release -DNOTEST=1
+
+      - name: Build
         run: |
-          cmake -B ${{github.workspace}}/pack -DCMAKE_BUILD_TYPE=Release -DNOTEST=1
           cmake --build ${{github.workspace}}/pack --config Release
           cpack --config ${{github.workspace}}/pack/CPackConfig.cmake -C Release -G ZIP
```
Comment (aliddell) [inline .github/workflows/build.yml]:
```suggestion
          cmake --build ${{github.workspace}}/build--config Release
          cpack --config ${{github.workspace}}/build/CPackConfig.cmake -C Release -G ZIP
```

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -43,13 +43,77 @@ jobs:
...
+        with:
+          submodules: true
+
+      - name: CMake
+        run: cmake -B ${{github.workspace}}/build -DCMAKE_BUILD_TYPE=Release -DNOTEST=1
+
+      - name: Build
+        run: |
+          cmake --build ${{github.workspace}}/pack --config Release
+          cpack --config ${{github.workspace}}/pack/CPackConfig.cmake -C Release -G ZIP
```
Comment (aliddell) [inline .github/workflows/build.yml]:
```suggestion
          cmake --build ${{github.workspace}}/build --config Release
          cpack --config ${{github.workspace}}/build/CPackConfig.cmake -C Release -G ZIP
```

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -72,9 +74,80 @@ jobs:
...
+
+      - name: Build
+        run: |
+          cmake --build ${{github.workspace}}/build --config Release
+          cpack --config ${{github.workspace}}/build/CPackConfig.cmake -C Release -G ZIP
+
+      - name: Test  # don't release unless tests pass
+        working-directory: ${{github.workspace}}/build
+        run: ctest -C Release -L anyplatform --output-on-failure
+
```
Comment (aliddell) [inline .github/workflows/release.yml]:
```suggestion
```

I see this is identical to the eGrabber release file, which highlights a mistake I've made. We've configured the build to skip tests, yet we're running ctest here🤦. We don't have any tests to run for this platform anyway.

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -72,9 +74,80 @@ jobs:
...
```
Comment (aliddell) [inline .github/workflows/release.yml]: Same story here, no tests to run.

```suggestion
```

**Review by aliddell — CHANGES_REQUESTED** [review]
Couple things to fix up.

File: .github/workflows/release.yml (hunk 3 above)
Comment (andy-sweet) [reply .github/workflows/release.yml]: Ah, good catch. In practice, it's possible some tests run (the test-acquire-core-libs-* ones), but they're not of interest, so we should still remove this anyway.

File: .github/workflows/build.yml (hunk 1 above)
Comment (andy-sweet) [reply .github/workflows/build.yml]: Ah, good catch. Thanks!

**Review by andy-sweet — COMMENTED** [review]
Thanks for the review!

**Review by aliddell — APPROVED** [review]

**Comment by andy-sweet** [comment]
Also brought in the simd.cmake fix and updated the changelog. Will tag the resulting merge commit with v0.1.1 assuming the workflows succeed.

**Pull request merged**